src/IndexService/ProductList/DefaultMysql/Dao.php

```diff
@@ -299,4 +299,13 @@ public function getLastRecordCount(): int
     {
         return $this->lastRecordCount;
     }
+
+    /**
+     * @param string $value
+     * @return string
+     */
+    public function quoteIdentifier($value)
+    {
+        return $this->db->quoteIdentifier($value);
+    }
```

Comment:
We should use type hints instead of php doc
```suggestion
    public function quoteIdentifier(string $value): string
    {
        return $this->db->quoteIdentifier($value);
    }
```

Reply:
changed, thanks for reviewing